End round stats

lmas opened the pull request:
- Fixes #473.
- Adds new SQL schema for storing end round stats.
- Stores said stats on end round.

This is part work of the apollo stats web page I'm currently working on. Yes I may have created a monster SQL query and yes, it was a bitch to debug.
I also fucked up the branch somehow so I couldn't save it on my own repo, @Kwask, after I had begun working on it.

Merry Xmas guys.

lmas commented:
Hold on, got some more TODOs after I made the web page:
- [x] Clean up some var names in the sql.
- [x] ~~Maybe remove the break time var~~ Nope looks good.
- [x] Add stats for kicked/banned/arrested guys.
- [x] Make new table for ai laws and tie them with the round id.
- [x] New table for antags and tie in with a round.

Kwask commented:
This doesn't modify any existing tables, correct? Those are always a pain to update :V

lmas commented:
Nope, just a new table(s).

lmas commented:
I lied. I updated the `death` table to add round_id to it. But I renamed it to `deaths` instead, so upgrading it will be just a simple `DROP TABLE death` and then just create the new table with the new name. 
A bonus is that deaths will now be reported on round end, only, and I can show more interesting stats on the web pages.

lmas commented on code/defines/procs/statistics.dm:
```diff
@@ -39,74 +39,6 @@ proc/sql_report_round_end()
-			var/err = query.ErrorMsg()
-			log_game("SQL ERROR during death reporting. Error : \[[err]\]\n")
-
-
```
Don't Repeat Yourself dammit! Keep things DRY!

lmas commented:
Merge whenever you're ready to update the database.

Kwask commented:
I'll plan on merging this tonight, when pop dies down a bit.
